### PR TITLE
[Functionality] Log app instance to the console

### PR DESCRIPTION
**Pull request by ev1stensberg**
Added logging statement, so the user knows when the app is up and running.

# Author Checklist

#### Changes address original issue? 
N/A

#### Unit tests included and/or updated with changes?
 Yes
#### Command line build passes?
 Yes

#### Changes have been smoke-tested? 
Yes

***

- Needs signed CLA agreement & review

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -77,7 +77,7 @@
 
     // Finally, open the HTTP server and log the instance to the console
     app.listen(options.port, function() {
-          console.log('Open MCT application running at localhost:' + options.port)
+        console.log('Open MCT application running at localhost:' + options.port)
     });
  
     }());
```
Comment (ev1stensberg) [inline app.js]: Do you want this indentation to be changed to none, or is it good?

File: app.js [hunk 2]
```diff
@@ -75,6 +75,9 @@
...
-    // Finally, open the HTTP server
-    app.listen(options.port);
-}());
\ No newline at end of file
+    // Finally, open the HTTP server and log the instance to the console
+    app.listen(options.port, function() {
+        console.log('Open MCT application running at localhost:' + options.port)
+    });
+ 
+    }());
```
Comment (VWoeltjen) [inline app.js]: There's some extra indentation on this last line (which closes the IIFE started on line 10)

**Comment by ev1stensberg** [comment]
Fixed now. I also sent a reply to `<andrew.k.henry@nasa.gov>`, should I expect a response there or do I need to send the email to someone else? It was about my thoughts around all these PR's and such.

**Comment by VWoeltjen** [comment]
> Fixed now. I also sent a reply to `<andrew.k.henry@nasa.gov>`, should I expect a response there or do I need to send the email to someone else? It was about my thoughts around all these PR's and such.

Confirmed with @akhenry that he received this email, I'll let him respond there.

Changes look good, we'll merge once everything is taken care of with the CLA. In the mean time, would you mind adding an [author checklist](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md#check-lists) for completeness?

### Reviewer Checklist

1. Changes appear to address issue? N/A &ast;
2. Appropriate unit tests included? N/A &dagger;
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y

&ast; No original issue; improvement is proposed as part of this pull request.
&dagger; Changes are external to client code and out of scope for unit testing.

**Comment by ev1stensberg** [comment]
@VWoeltjen Done.

**Comment by ev1stensberg** [comment]
@VWoeltjen sent the signed CLA to @akhenry `s email, let me know if there anything else you need!

**Comment by ev1stensberg** [comment]
FWIW-- we also could do

**`console.log("listening on port: %s and app is running on localhost:%s", option.args)`**

**Comment by akhenry** [comment]
CLA received, merging.

**Pull request merged**